crates/editor/src/hunk_diff.rs

```diff
@@ -608,7 +605,7 @@ impl Editor {
                                                                 move |menu, _| {
                                                                     menu.context(focus.clone())
                                                                         .action(
-                                                                            "Discard All",
+                                                                            "Discard All Changes in File",
```

Comment:
I don't know that we should specify "in file" here, as in the proposed changes editor it doesn't actually correspond to a file.

I think "Discard All" is clear enough, IMO, and it also matches what Sublime Merge does:

<img width="263" alt="Screenshot 2024-10-02 at 12 44 54 PM" src="https://github.com/user-attachments/assets/195af6e2-17c4-4920-9897-2c9e82386380">

Reply:
I think Sublime Merge's design as a whole (with the header that has the "Discard All" and "Stage All" buttons sticky-positioned and above all changes) is what makes the label clearer. I don't feel like our case is exactly the same, because we're exposing this global-affecting action in each individual hunk, which is why I thought of specifying that a bit more with the label. Sublime Merge don't let you discard a whole set of changes via a specific hunk—they have this hierarchy of hunk → file → all changes.

<img width="700" alt="Screenshot 2024-10-02 at 6 53 56 PM" src="https://github.com/user-attachments/assets/25b9e800-6ac0-4c52-b7ad-31957d45dc35">

I'm happy to revert it, though, particularly if "...in File" is technically inaccurate. I just felt like giving an extra level of redundancy here with the copywriting because the design can cause some hesitation—like, someone could ask "Discard all... what? lines in this hunk?" or something.

Reply:
We could make it "Discard All Changes" or "Discard All Hunks".

But yea, I don't think we should include "in File" since it's not always going to be accurate.